packages/js/client/src/__tests__/core/resolveUri.spec.ts

```diff
@@ -708,50 +678,28 @@ describe("resolveUri", () => {
         },
       },
       {
-        uriResolver: "CacheResolver",
-        sourceUri: ipfsUri,
-        result: {
-          uri: ipfsUri,
-          wrapper: false,
-        },
-      },
-      {
-        uriResolver: "PluginResolver",
-        sourceUri: ipfsUri,
-        result: {
-          uri: ipfsUri,
-          wrapper: false,
-        },
-      },
-      {
-        uriResolver: "ExtendableUriResolver",
         sourceUri: ipfsUri,
+        uriResolver: "CacheResolver",
         result: {
           uri: ipfsUri,
           wrapper: true,
-          implementationUri: new Uri("wrap://ens/ipfs-resolver.polywrap.eth"),
         },
       },
     ]);
   });
 
-  it("can resolve wrapper with redirects", async () => {
-    await runCLI({
-      args: ["build"],
-      cwd: `${GetPathToTestWrappers()}/wasm-as/interface-invoke/test-interface`,
-    });
-
+  it("can resolve wrapper an ENS wrapper whose IPFS URI has been redirected", async () => {
```

Comment:
Is this well written?

Reply:
Kinda, I'll tackle this with a PR for "remove test-env from resolveUri tests"